README.md

```diff
@@ -0,0 +1,34 @@
+# ansible_cephadm
+
+This role installs [cephadm](https://docs.ceph.com/en/latest/cephadm/) and Docker on Debian.
```

Comment:
There is no task for installing docker

Reply:
Right, I removed that because we don't use it.